docs/source/api_references.rst

```diff
@@ -87,7 +87,7 @@ Loggers API
 
     base
     comet
-    csv_logs
+    csv
```

Comment:
This PR needs a deprecation. You can see how it was done in https://github.com/PyTorchLightning/pytorch-lightning/pull/11227 (for example) and mimic the changes.